### PR TITLE
Added some Pokemon and updated some styling 

### PR DESCRIPTION
**Pull request by KingFruit85**
Learning git forking and pull requests, hope that's ok!

### CODE REVIEW

File: index.html
Comment (HedgehogHolly) [inline index.html]: As you've added the text for each of the types, you could get rid of the tool tips I had made as theres not much point in them now.

File: style.css
Comment (HedgehogHolly) [inline style.css]: When you add a font, its always a good idea to add a back-up incase the computer doesn't have it like 'font-family: helvetica, sans-serif, Arial;'

File: style.css
Comment (HedgehogHolly) [inline style.css]: For readability/accessibility, some of the types would work better if the text was in black, like Fairy and Normal, but some will need white text like Fighting and Poison. It might be a good idea to create two classes one for white type and one for black type.

**Review by HedgehogHolly — COMMENTED** [review]
Some suggestions 🐧

File: index.html
Comment (KingFruit85) [reply index.html]: I think I have done this and the commit appears to appear in the pull request?

Comment (HedgehogHolly) [reply index.html]: Perfect, I can see you've removed it 😄

File: style.css
Comment (KingFruit85) [reply style.css]: I didn't know about that, cool, updated!

File: style.css
Comment (KingFruit85) [reply style.css]: Done, let me know what you think :)

File: style.css
Comment (HedgehogHolly) [reply style.css]: All good! 👍

File: style.css
Comment (HedgehogHolly) [reply style.css]: Awesome, I think bug could maybe be black as well and fighting maybe would be better as white 🤔 This is a good resource for checking the accessibility of colours with background and foreground - https://webaim.org/resources/contrastchecker/ getting it to at least AA standards is always good. 

You can mess around with that but apart from that the code itself looks good!

Comment (KingFruit85) [reply style.css]: Updated all font/background colours so they meet AAA standard on https://webaim.org/resources/contrastchecker/

Comment (HedgehogHolly) [reply style.css]: Amazing! 👏 🏆

**Review by HedgehogHolly — APPROVED** [review]
Woop Woop! 💃

File: style.css
Comment (HedgehogHolly) [inline style.css]: Good idea :)

**Comment by KingFruit85** [comment]
Committing changes to base branch.

**Comment by KingFruit85** [comment]
I can't merge as I don't have write access to your repo :p @HedgehogHolly

**Comment by HedgehogHolly** [comment]
👏

**Pull request merged**
